R/makeClusterTask.R

```diff
@@ -93,10 +93,9 @@ makeClusterTask = function(id, data, target, cluster.cols = NULL, method = "clus
   #target will be checked within GetDataType
   #check cluster.cols
   if (!is.null(cluster.cols)) {
-    assertCharacter(cluster.cols, min.len = 1, max.len = 10)
+    assertCharacter(cluster.cols, min.len = 2, max.len = 10)
```

Comment:
@ptl93 
I'm not 100% sure but cluster.cols should always be at least of length 2 right?

Reply:
No. In this case cluster.cols is a named vector and therefore contains two values already even if the dimension is 1.

example: 
cluster.cols = c("a" = "b"). Hence we map a to b in the clustering